### PR TITLE
RYA-491 Repackaged reusable Mongo DB test code into a project named '…

### PR DESCRIPTION
**Pull request by kchilton2**
…rya.test.mongo'.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -381,6 +458,26 @@ under the License.
                 <artifactId>rya.indexing.pcj</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.rya</groupId>
+                <artifactId>rya.kafka.connect.parent</artifactId>
```
Comment (ejwhite922) [inline pom.xml]: Don't think these belong in this PR.

Comment (kchilton2) [reply pom.xml]: Yep, not sure how that got in there.

File: dao/mongodb.rya/src/test/java/org/apache/rya/mongodb/MongoDBRyaDAO2IT.java [hunk 2]
```diff
@@ -40,11 +40,11 @@
 /**
  * Integration tests the methods of {@link MongoDBRyaDAO}.
  */
-public class MongoDBRyaDAO2IT extends MongoITBase {
+public class MongoDBRyaDAO2IT extends MongoRyaITBase {
```
[the remark this answers is not in the document]
Comment (kchilton2) [reply dao/mongodb.rya/src/test/java/org/apache/rya/mongodb/MongoDBRyaDAO2IT.java]: I'm not messing with classes that already exist except to deal with my repackaging.

File: dao/mongodb.rya/src/test/java/org/apache/rya/mongodb/aggregation/PipelineQueryIT.java [hunk 3]
```diff
@@ -423,28 +423,28 @@ public void testRequiredTimestamp() throws Exception {
...
-        Multiset<BindingSet> solutions = HashMultiset.create();
+        final Multiset<BindingSet> solutions = HashMultiset.create();
         CloseableIteration<BindingSet, QueryEvaluationException> iter = pipelineNode.evaluate(new QueryBindingSet());
         while (iter.hasNext()) {
             solutions.add(iter.next());
         }
         Assert.assertEquals(expectedSolutions, solutions);
         // Extend the pipeline by requiring a future timestamp (should produce no results)
-        long delta = 1000 * 60 * 60 * 24;
+        final long delta = 1000 * 60 * 60 * 24;
```
[the remark this answers is not in the document]
Comment (kchilton2) [reply dao/mongodb.rya/src/test/java/org/apache/rya/mongodb/aggregation/PipelineQueryIT.java]: I'm not messing with classes that already exist except to deal with my repackaging.

**Comment by kchilton2** [comment]
Updated.

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-rya-master-with-optionals-pull-requests/744/

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-rya-master-with-optionals-pull-requests/745/

**Pull request closed without merging**